dockers/skylab/loom-output/create_loom_optimus.py

```diff
@@ -430,12 +434,30 @@ def main():
     )
 
     parser.add_argument(
-        "--sample_id",
-        dest="sample_id",
-        default="Unknown sample",
+        "--input_id",
+        dest="input_id",
+        required=True,
         help="the sample name in the bundle",
     )
 
+    parser.add_argument(
+        "--input_name",
+        dest="input_name",
+        help= "sequencing_input.biomaterial_core.biomaterial_id in HCA metadata, defined by the user",
+    )
+
+    parser.add_argument(
+        "--input_id_metadata_field",
+        dest="input_id_metadata_field",
+        help= "sequencing_process.provenance.document_id: [UUID] defined by the user",
```

Comment:
If it's a UUID it's not really defined by the user for HCA, but maybe you mean defined by user when the pipeline is run? I like that this documentation notes what field is used in HCA given these fields are basically entirely for HCA, but I wonder if they should also give other users more context? Like for input_id_metadata_field it could also say "the name of the metadata field used for the input_id"

Is this saying that the way it will appear in the outputs is input_id_metadata-field: [UUID from input_id] ? 

Like say the `input_id_metadata_field` is called `my_favorite_metadata` and the UUID in `input_id` was `ABC123`

Would the loom say `my_favorite_metadata:ABC123` ?